bigbluebutton-html5/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "start": "cd app;./start.sh",
     "preinstall": "npm install grunt-cli",
-    "test": "cd app;JASMINE_BROWSER=PhantomJS JASMINE_MIRROR_PORT=3000 meteor run --test",
+    "test": "node_modules/.bin/wdio ./tests/webdriverio/wdio.conf.js",
```

Comment:
You dont need the `node_modules/.bin` node already add all bin's to the PATH while using node-scritps.